deepspeed/launcher/runner.py

```diff
@@ -74,7 +74,7 @@ def parse_args(args=None):
                         type=str,
                         default="",
                         help='''Specify hardware resources to NOT use during execution.
-                        Mutually exclusive with --include. Resource formatting
+                        Mutually exclusive with --exclude. Resource formatting
```

Comment:
@oushu1zhangxiangxuan1 I believe this is correct as it was, the --exclude option is mutually exclusive, as in it is incompatible with the --include option, so you cannot use both.  So in the --exclude option should mention the include option.

Reply:
@loadams Thanks, you are right. I misunderstood it‘s meaning.